packages/elements-core/src/components/Docs/HttpService/AdditionalInfo.tsx

```diff
@@ -0,0 +1,45 @@
+import { Panel } from '@stoplight/mosaic';
+import { IHttpService } from '@stoplight/types';
+import React from 'react';
+
+import { MarkdownViewer } from '../../MarkdownViewer';
+
+const spdxLicenseList = require('spdx-license-list');
+
+interface AdditionalInfoProps {
+  termsOfService?: string;
+  contact?: IHttpService['contact'];
+  license?: IHttpService['license'];
+}
+
+export const AdditionalInfo: React.FC<AdditionalInfoProps> = ({ termsOfService, contact, license }) => {
+  const contactLink =
+    contact?.name && contact?.url
+      ? `[Contact ${contact.name}](${contact.url})`
+      : contact?.email
+      ? `[Contact ${contact.name || contact.email}](mailto:${contact.email})`
+      : '';
+
+  //use spdx to look up url for license identifier if available
+  const licenseUrl =
+    license?.url || (spdxLicenseList[`${license?.identifier}`] && spdxLicenseList[`${license?.identifier}`].url);
+
+  const licenseLink = license?.name && licenseUrl ? `[License: ${license.name}](${licenseUrl})` : '';
```

Comment:
I don't think this works if `license.identifier` is present but not `license.name`. You could look up the `name` from `spdx-license-list` the same way you look up the `url`.

Reply:
I take that back. I see `name` is "REQUIRED" by the OpenAPI 3.1 spec. So we should probably force customers to use it correctly 😆

Reply:
p.s. The reason I think it's a bad design decision on OpenAPI's part is it means this is valid:

```json
{
  "name": "BSD 3-Clause",
  "identifier": "Apache-2.0"
}
```

and heavens knows what that means, except fun for lawyers. Particularly in our docs rendering, unless they click on the link, they wouldn't even notice the URL points to a different license than the name field.

Reply:
@wmhilton maybe I am confusing myself, but I think that I intentionally made it so that it won't show anything if there's not a name AND a link (or a name AND an identifier to look up the link) based on the fact that Spectral will yell at someone if they don't include the name, so the onus is on the user to add one. 

Also, agree with why you think it's bad design! When I was testing it out, I was using random names with real identifiers and thought that seemed weird :D 

Do you think I should alter it?

Reply:
> Do you think I should alter it?

Nope, let's leave it as is. I may disagree with the standard but if everyone writes code based on what they _wish_ the standard said, you end up undermining the standard. (And you get really awkward behavior like HTML parsers where some badly written HTML works in some browsers but is broke in others.) The standard says `name` is required, Spectral will yell if they don't include the name, Elements should treat name as required too then.